test/parallel_api/ranges/zip_view.pass.cpp

```diff
@@ -40,6 +40,26 @@ main()
 
     //check access
     EXPECT_TRUE(::std::get<0>(z[2]) == 'g', "wrong effect with zip_view");
+
+    const size_t max_int32p2 = (size_t)::std::numeric_limits<int32_t>::max() + 2UL;
+    ::std::vector<char> large_data(max_int32p2);
+    ::std::vector<char> large_keys(max_int32p2);
```

Comment:
Allocating 4GB of memory twice is quite a lot, and it may take quite some time as well since all the elements should be initialized. I would recommend an approach that requires less space, for example, using views::iota and views::transform to map big indexes to a reasonably small data container.

Reply:
I adjusted the test to reuse existing small vectors as you describe.